contracts/base/Polymorph.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity ^0.8.0;
 import "./IPolymorph.sol";
 import "@openzeppelin/contracts/utils/math/SafeMath.sol";
 import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
-import "@openzeppelin/contracts/utils/Counters.sol";
+import "../lib/Counters.sol";
```

Comment:
The only reason for usage of `Counters` over a plain `uint256` that gets incremented is that the Counters cannot be assigned. As we need the assignment, Id say - ditch the `Counters` and just use `uint256` that gets incremented

Reply:
Done. 
Should I also remove the 'AutoId' part from 'ERC721PresetMinterPauserAutoId', as we don't use Counters anymore ?
Also, should we remove the usage of SafeMath for uint256, as we migrated contracts from ^0.7 to ^0.8 ?

Reply:
Correction: 
After deeper diving, ERC721PresetMinterPauserAutoId should stay as it is, otherwise we'll break setBaseURI() funnctionality because _baseTokenURI is private.
I have also removed SafeMath and added a specific version to solidity (0.8.13) cc @strenev